### PR TITLE
 Fixes #14396: Does not highlight dot based output

### LINKED ISSUE #14396: Graph commands `ag*` output are effected by visual highlighting
### Expected behavior
When using a graph command like `agCd` to output graphviz dot format it should contain valid dot syntax. Even if you have turned on highlighting in the `V`isual mode with `/`

### Actual behavior
The output will be tainted with color control chars if the graph output text contains matching highlight string data.

### Steps to reproduce the behavior 
- Open a binary with r2 and enter visual mode
- use `/di` to highlight text (as if you were looking for all edi/rdi registers)
- use `agCd > test.dot` to dump the callgraph into a dot file
- the dot file will contain color control characters which make it invalid

### Additional Logs, screenshots, source-code,  configuration dump, ...
image

### PR DESCRIPTION
**Pull request by deepakchethan**

### CODE REVIEW

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -7369,6 +7369,7 @@ static void cmd_agraph_print(RCore *core, const char *input) {
 
 static void cmd_anal_graph(RCore *core, const char *input) {
 	core->graph->show_node_titles = r_config_get_i (core->config, "graph.ntitles");
+	r_cons_enable_highlight(false);
```
Comment (XVilka) [inline libr/core/cmd_anal.c]: add a space before `(`.

File: libr/cons/cons.c [hunk 2]
```diff
@@ -1536,6 +1541,10 @@ R_API void r_cons_highlight(const char *word) {
 		strlen (inv[1])
 	};
 
+        if (!I.enable_highlight) {
+                r_cons_enable_highlight(true);
```
Comment (XVilka) [inline libr/cons/cons.c]: add a space before `(`.

**Comment by XVilka** [comment]
@deepakchethan can you rebase please?

**Comment by deepakchethan** [comment]
It is already the latest, I believe. When I try rebase it says Already up to date.

**Pull request merged**